pkg/repo/mysql/services_integration_test.go

```diff
@@ -0,0 +1,218 @@
+package mysql
+
+import (
+	"context"
+	"log"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/brianvoe/gofakeit/v6"
+	"github.com/google/uuid"
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/assert"
+	"go.expect.digital/translate/pkg/model"
+	"go.expect.digital/translate/pkg/repo"
+	"go.expect.digital/translate/pkg/tracer"
+)
+
+var repository *Repo
+
+func TestMain(m *testing.M) {
+	ctx := context.Background()
+
+	viper.SetEnvPrefix("translate")
+	viper.SetEnvKeyReplacer(strings.NewReplacer(".", "_"))
+
+	viper.AutomaticEnv()
+
+	tp, err := tracer.TracerProvider()
+	if err != nil {
+		log.Panicf("set tracer provider: %v", err)
+	}
+
+	repository, err = NewRepo(WithDefaultDB(ctx))
+	if err != nil {
+		log.Panicf("create new repo: %v", err)
+	}
+
+	code := m.Run()
+
+	repository.db.Close()
+
+	if err := tp.Shutdown(ctx); err != nil {
+		log.Panicf("tp shutdown: %v", err)
+	}
+
+	os.Exit(code)
+}
+
+func randService() *model.Service {
+	return &model.Service{
+		Name: gofakeit.FirstName(),
+		ID:   uuid.New(),
+	}
+}
+
+func Test_SaveService(t *testing.T) {
+	t.Parallel()
+
+	service := randService()
+	err := repository.SaveService(context.Background(), service)
+
+	assert.NoError(t, err)
+}
```

Comment:
Is it enough to assert that there is no error after calling `repository.SaveService()` ?
Or should we expand these tests with additional, 
```go
actual, err := repository.LoadService(ctx, service.ID)
...
assert.Equal(t, "loaded from repo", "created randService()")
```

I like It because then we can be 100% sure that everything is working as expected, especially this is for `UpdateService Test`. But I don't like it because we are creating spaghetti of repo methods, in almost all tests for example:
`Test_SaveService = SaveService + LoadService`

Reply:
I have come to the conclusion that I should assert that the values obtained from the repository match the expected values.